### PR TITLE
WS-60 old fix that did not get merged

### PR DESCRIPTION
**Pull request by vercasson**

### CODE REVIEW

File: src/Map.js [hunk 1]
```diff
@@ -189,13 +189,10 @@ export default class Map extends Component {
     const loop = async () => {
       const { api, locationID, tags } = this.props;
       const floorID = STREAM_ALL_FLOORS;
```
Comment (vercasson) [inline src/Map.js]: filtering happens within `TagListOverlay`

Comment (vercasson) [reply src/Map.js]: I testing the crap out of this today, and back when I made the fixes.

File: src/FloorOverlay.js [hunk 2]
```diff
@@ -168,9 +168,8 @@ class FloorOverlay extends Component {
 FloorOverlay.propTypes = {
   toggleFloorOverlay: PropTypes.func.isRequired,
   currentFloorID: PropTypes.string.isRequired,
-  floors: PropTypes.object.isRequired,
-  selectFloorByID: PropTypes.func.isRequired,
-  closeFloorOverlay: PropTypes.func.isRequired
+  floors: PropTypes.array.isRequired,
+  selectFloorByID: PropTypes.func.isRequired
```
Comment (wavebeem) [inline src/FloorOverlay.js]: was closeFloorOverlay not used any more?

Comment (vercasson) [reply src/FloorOverlay.js]: Correct, `toggleFloorOverlay` replaced that

**Pull request merged**